introduction.md

```diff
@@ -0,0 +1 @@
+Of course I talk to myself, Because sometimes I need expert advice.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.